### ISSUE TITLE
Odoo 16: Dropdown menu for Document Layout not available.  Can't change layout.

### ISSUE THREAD

**Issue by sugapablo** [odoo/odoo#137207]
Impacted versions: Odoo 16.0+e (Enterprise Edition)

Steps to reproduce: In Developer Mode, go to Settings and scroll down to "Document Layout".

Current behavior: Document Layout is simply a link.  See screenshot from runbot:

<img width="1183" alt="Screen Shot 2023-10-01 at 9 27 19 PM" src="https://github.com/odoo/odoo/assets/10548725/a44b221f-14d9-4f37-81d3-c499fbc8248d">

Expected behavior: Document Layout should be a drop down field where you can select which layout to use (including custom layouts).  See screenshot from Odoo 15 runbot:

<img width="1182" alt="Screen Shot 2023-10-01 at 9 28 45 PM" src="https://github.com/odoo/odoo/assets/10548725/4c9b0d5f-be55-48d5-a9a2-80cd4f174ac1">

If this is a purposeful change, how can someone change layout to a custom layout here?

**Comment by bbonfim** [comment]
Also having the same issue. After the latest release, we cannot set our custom layout anymore.

**Comment by sugapablo** [comment]
> hi , this seems to be after this commit: [03cffc4](https://github.com/odoo/odoo/commit/03cffc460e620639db6dbe132aa0be073a6396f5)
> 
> as of now, users can change the layout from the pop up by clicking the Configure Document Layout and selecting layout
> 
> ![Screenshot from 2023-10-02 22-51-52](https://user-images.githubusercontent.com/27989791/272071957-068d9620-b6f3-4b48-999e-5f6d46470d48.png)
> 
> thanks

But then how can we add custom layouts now?  Are Light, Boxed, Bold, and Striped hardcoded?

**Comment by bbonfim** [comment]
> > hi , this seems to be after this commit: [03cffc4](https://github.com/odoo/odoo/commit/03cffc460e620639db6dbe132aa0be073a6396f5)
> > as of now, users can change the layout from the pop up by clicking the Configure Document Layout and selecting layout
> > ![Screenshot from 2023-10-02 22-51-52](https://user-images.githubusercontent.com/27989791/272071957-068d9620-b6f3-4b48-999e-5f6d46470d48.png)
> > thanks
> 
> But then how can we add custom layouts now? Are Light, Boxed, Bold, and Striped hardcoded?

exactly! We have highly customized headers and footers that we ported from our older ERP to odoo and now we just can't have it anymore? I mean, it is already set in production thankfully, but every time I create a new branch in odoo sh I have to change this layout to test it properly but I can't anymore.
Somebody gives us a proper solution please! 

Can we set it directly in the database?
Should we update the template manually using the interface? (there is a huge warning pointing against that, for obvious reasons)
Will it be the same in Odoo 17?

Appreciate any feedback here! and thanks @sugapablo for raising the issue!

**Comment by moneebalalfi** [comment]
Hey guys .. @bbonfim @sugapablo 
Anyone solve this issue ? 

Thanks :)

**Comment by sswapnesh** [comment]
This field (external_report_layout_id) is related field from company (res.company) so, one can just display this field on company form view and change it from there.

**Comment by bbonfim** [comment]
@sswapnesh Indeed! that worked like a charm! thanks a lot!
